reswift-jobs/reswift-jobs/JobDocument.swift

```diff
@@ -27,6 +27,7 @@ class JobDocument: NSDocument {
         let storyboard = NSStoryboard(name: NSStoryboard.Name("Main"), bundle: nil)
         let windowController = storyboard.instantiateController(withIdentifier: NSStoryboard.SceneIdentifier("JobWindowController")) as! JobWindowController
         self.addWindowController(windowController)
+        windowController.store = self.store
```

Comment:
doesn't need `self`. I guess you can leave for consistency with the rest of the file, or just remove all `self` where they are not needed, unless to fix ambiguity for the compiler.

Reply:
Yeah, it's for consistency. I'm basing this project off of the original TODO example app, with the idea of doing a minimum amount of coding style change.